### ISSUE TITLE
Unavoidable Hypothermia death

### ISSUE THREAD

**Issue by Jackiechun117** [Glitchfiend/ToughAsNails#407]
For some reason after several hours of gameplay on two saves my character will suddenly be unable to gain warmth from heat sources. Up until this point the mechanic works fine, Vanilla heats sources will work. Magma blocks, furnaces, fire, lava. Then suddenly My temperature will not stop dropping even while standing right next to lava or surrounded by working furnaces and magma blocks.

**Comment by Adubbz** [comment]
You can use /tan tempinfo to show you why your temperature is falling. We can't really determine whether it is a bug or not without it

**Comment by Kendallfish** [comment]
I'm having this problem today too. I've played for hours today and yesterday, but when I logged on tonight my temperature kept dropping to damaging hypothermia, even though I was next to heating coils and wearing all wool clothes. The temperature algorithm has always given me a big headache, but now it is truly not working. I've also noticed that after the early spring season, the seasons reset to autumn. I'm on my hundredth day in this world now, and the leaves are red and brown.
